op-e2e/setup.go

```diff
@@ -638,18 +650,25 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 			if err != nil {
 				return nil, err
 			}
+
 			ethClient = l2Geth
 		} else {
 			if len(cfg.GethOptions[name]) > 0 {
 				t.Skip("External L2 nodes do not support configuration through GethOptions")
 			}
+
+			if name != RoleSeq && !cfg.DisableTxForwarder {
+				cfg.Loggers[name].Warn("External L2 nodes do not support `RollupSequencerHTTP` configuration. No tx forwarding support.")
```

Comment:
This could `panic` to prevent misconfiguration to ensure that people don't think they are doing something that they cannot, not sure if that would break many tests

Reply:
Since `DisableTxForwarder` is `true` by default, this would panic by default as well hence the warning instead of erroring out.

It felt more appropriate to enable tx forwarding by default instead of explicitly enabling it. do you think otherwise?